karma.conf.js

```diff
@@ -0,0 +1,46 @@
+module.exports = function (config) {
+  config.set({
+
+    port: 9876,
+
+    logLevel: config.LOG_INFO,
+
+    autoWatch: true,
+    singleRun: false,
+    colors: true,
+
+    plugins: [
+      'karma-jasmine',
+      'karma-sinon',
+      'karma-spec-reporter',
+      'karma-chrome-launcher',
+      'karma-firefox-launcher',
+      'karma-browserify'
+    ],
+
+    browsers: ['Chrome'],
```

Comment:
The test always run in CLI with Karma, we can make Phantom default if it's on 2.0 but I thought I had a CLI override in place to specify the browser. If not I'll add it.